### PR TITLE
Message validation

### LINKED ISSUE #6: Message validations
As a user, I want to be the only one that can edit or delete my messages, nobody else can do that.
- Allowed images should be of the following formats: png, jpg, jpeg
  - In case the user tries with another format it will show an error about the image
- Current user shouldn't be able to modify or delete another user's messages.

### PR DESCRIPTION
**Pull request by GregVaz**
Validates format image and user property message actions

### CODE REVIEW

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -8,9 +8,16 @@ class MessagesController < ApplicationController
...
-      format.js
+    if @message.valid?
+      respond_to do |format|
+        format.html
+        format.js
+      end
+    else
+      @message.errors.full_messages.each do |error_message|
+        flash[:error] = error_message
+      end
```
Comment (ese-varo) [inline app/controllers/messages_controller.rb]: You could write it in just one line: 
```suggestion
      @message.errors.full_messages.each  { |error_message| flash[:error] = error_message }
```

**Review by ese-varo — APPROVED** [review]
Great work Gregorio, I just left a comment :rocket:

File: app/controllers/messages_controller.rb (hunk 1 above)
Comment (GregVaz) [reply app/controllers/messages_controller.rb]: Correct :+1:

**Review by LoginALee — APPROVED** [review]
Great work Goyo!

**Review by jvargas98 — APPROVED** [review]

**Pull request merged**